tools/sigh

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-exec node $(dirname $0)/sigh.js "$@"
+node_modules/.bin/tsc -p tools && exec node build/sigh.js "$@"
```

Comment:
can you use ts-node?

https://github.com/TypeStrong/ts-node

Reply:
Possibly; it does work, but prior to this CL ts-node doesn't appear to be needed for the standard Arcs build - I'm not sure why it's in the top-level package.json in the first place.

Seems like a waste to pull in another dependency just for sigh. Note that ts-node does make the execution faster, so if we stick to using plain node I'll add a simple timestamp check to avoid unnecessary recompilation.